Neos.Flow/Classes/Security/Channel/HttpsInterceptor.php

```diff
@@ -48,5 +49,6 @@ public function __construct(
      */
     public function invoke()
     {
+        return true;
```

Comment:
Hello dear reviewer, are the changes here correct?

Reply:
Well… I think this interceptor was never properly implemented. 😬 So it might even be totally irrelevant.

Reply:
Yeah seems not properly implemented, I am fine with this change for now, _maybe_ a TODO / FIXME ala "throw away or finish"